RetrospectiveExtension.Frontend/package.json

```diff
@@ -84,12 +88,18 @@
     "webpack-dev-server": "^4.7.2"
   },
   "jest": {
+    "testPathIgnorePatterns" : [
+      "./components/__tests__/setupTests.tsx"
+    ],
     "transform": {
       "^.+\\.(js|ts|tsx|jsx)$": "ts-jest"
     },
     "transformIgnorePatterns": [
       "node_modules/(?!(azure-devops-ui|azure-devops-extension-sdk|azure-devops-extension-api)/)"
     ],
+    "setupFiles": [
```

Comment:
this will run all of the shared mocks once at the beginning of running all of the tests.